oldScripts/gradescope_to_spreadsheet.py

```diff
@@ -155,8 +155,10 @@ def retrieve_grades_from_gradescope(gradescope_client, assignment_id = ASSIGNMEN
 
 @deprecated
 def initialize_gs_client():
+    with open('gs_credentials.json', 'r') as credentials_file:
```

Comment:
Please load credentials from a .env file instead because this is sensitive data. Add a .gitignore with .env added to the .gitignore. Everything else is good. For an example, look at the api folder in this repo

Reply:
Thanks for pointing me to the example in the api folder, Mehul! This is the first time I am seeing a python script access environment variables. Glad to learn!